src/loaders/zarr_utils/ChunkPrefetchIterator.ts

```diff
@@ -37,7 +46,7 @@ export default class ChunkPrefetchIterator {
   constructor(
     chunks: TCZYX<number>[],
     tzyxMaxPrefetchOffset: TZYX,
-    tzyxNumChunks: TZYX,
+    tczyxChunksPerDimension: TCZYX<number>[],
```

Comment:
Not sure I understand what this arg contains. Would `tczyxChunksPerSource` be more appropriate here?  tczyx are dimensions...